melatonin/helpers/component_helpers.h

```diff
@@ -49,6 +49,19 @@ namespace melatonin
             return juce::String ("Editor: ") + editor->getAudioProcessor()->getName();
         }
 #endif
+        else if (c && c->isAccessible() && c->getAccessibilityHandler() && !c->getAccessibilityHandler()->getTitle().isEmpty())
+        {
+            // If a widget has an accessible name, prefer that to the internal
+            // name since it is user facing in a screen reader
+            auto acctitle = c->getAccessibilityHandler()->getTitle();
+            auto nm = c->getName();
+            if (nm != acctitle &&  !nm.isEmpty())
+            {
+                // but if i also have an internal name, dont' suppress it
+                acctitle += "(name=" + nm + ")";
```

Comment:
I'll check this out locally and toy with it. From reading the code, I think will lead to ugly names in the tree like like `Pitch Stiffness(pitchStiffness)` for people like me who use component's `setName` widely. I think it's good thinking to prefer the accessible name for tree display but perhaps we should always display a component's name as a discrete property (like we do with `Class`). 

Sidebar: I can't help but feel like this is in part a JUCE issue / historical artifact that component has both name / title. There's some conventions missing so I'm guessing devs are all over the place in terms of their implementations. In my setups, a component's name is a "key" that's used to look up its "UI friendly name" from data which is then used in the UI, can be used for accessibility, can be swapped out with translations, etc....

Reply:
Yeah I agree with your sidebar

Putting the name after the class in the properties list and backing out this change is a good idea

And yes having an accessibility top level area would also solve a lot of the other things we want to do but was a bigger change so wasn't sure if you were up for it. Then we could do things like 'next- and previous- focus target' and the action list also.

Reply:
Also: I found out from using inspector that I don't have a name on *any* of my components! 

One of the things I think juce wanted to avoid with name/title by the way is if users had given their components names like 'dumbSwitchToUnbreakStuff' in code, exposing that as user display for screen reader users only was a bit risky. That's also why I chose the screen reader name to beat the accesible name since it is in the end user UX.

Reply:
> One of the things I think juce wanted to avoid 

Amazingly get/setName seem to be... 17 years old according to git blame. I'm guessing they were mainly used as internal identifiers?

> Then we could do things like 'next- and previous- focus target' and the action list also.

I'm into it! Let's make it its own panel. 

Some of the panel / header logic is a bit messy if I remember (the headers overlay the panels as they can contain action buttons when collapsed. And I think PROPERTIES is setup to be the last panel, would need some work if we wanted to plop accessibility afterwards. In short, feel free to uhh.. nope out of that work if it gets finicky, I don't mind getting in there.

Reply:
Ha I'll take a peek today and if its impossible i'll revert but otherwise will update this PR once I have it in